api/spec/src/app/main.tsp

```diff
@@ -0,0 +1,18 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi";
+import "@typespec/openapi3";
+
+import "..";
```

Comment:
If the app is imported in parent's main, I don't think this is necessary.

Reply:
It breaks without it